### DISCUSSION TITLE
Allow for word wrap in file editor

### DISCUSSION THREAD

**Discussion by cowtrix** [community/community#5009] [Other Feature Feedback, Questions, & Ideas]
Editing files through the GitHub website is a great feature. However, the lack of an ability to toggle word wrap makes it difficult to edit text files with long paragraphs, especially on mobile. Having a checkbox to toggle the word wrap of the file edit text field would be a great option.

**Comment by auwsom** [comment]
It looks like there are No wrap and Soft wrap options in the editor now. However, they are not available on the Gist editor. This leads me to ask why they are not using the same basic editor features.
